python/py-requests-cache/Portfile

```diff
@@ -4,8 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           python 1.0
 
-github.setup        reclosedev requests-cache 6717a336430eb7e6fe17e75c5517f3076ca85b42
-version             0.5.2
+version             0.8.1
 revision            0
 name                py-requests-cache
```

Comment:
please add the `name` field above the `version` (not strictly required, but conventional)